### ISSUE TITLE
Generate random string values in templates

### ISSUE THREAD

**Issue by royallthefourth** [sj14/dbbench#6]
This tool would be more useful to me if it was capable of generating things like names and addresses to benchmark on more realistic data.

Would you accept a PR adding a fake data generation library dependency and some new template functions to go with it?

**Comment by sj14** [comment]
Of course, I can't promise to merge it before seeing it, but generally spoken, that would be great! I also thought about adding a fuzzer library dependency for this.

**Comment by royallthefourth** [comment]
Great, I'll see if I can come up with something decent this weekend
